package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "spruce",
-  "version": "3.0.134",
+  "version": "3.0.135",
```

Comment:
You need to merge main into the base branch so that this isn't included as part of the diff.